torch/hub.py

```diff
@@ -340,23 +344,24 @@ def load(repo_or_dir, model, *args, **kwargs):
             ``source = 'local'``.
         model (string): the name of a callable (entrypoint) defined in the
             repo/dir's ``hubconf.py``.
-        *args (optional): the corresponding args for callable :attr:`model`.
-        source (string, optional): ``'github'`` | ``'local'``. Specifies how
-            ``repo_or_dir`` is to be interpreted. Default is ``'github'``.
+        *args (optional): the corresponding args for callable ``model``.
+        source (string, optional): 'github' or 'local'. Specifies how
```

Comment:
nit: I think keeping double ticks around `'github'` and `'local'` would make it easier to distinguish them.

Reply:
For such strings I tend to find that the grey highlighting induced by the backquotes is a bit overkill sometimes, especially when there are many of them, it can be overwhelming to the eye. Since they are already delimited with single quotes, they still stand, but in a smoother way IMHO. No strong opinion though.